xml/System.Reflection.Metadata/ManifestResource.xml

```diff
@@ -102,8 +102,8 @@
         <ReturnType>System.Reflection.Metadata.EntityHandle</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary><see cref="T:System.Reflection.Metadata.AssemblyFileHandle" />, <see cref="T:System.Reflection.Metadata.AssemblyReferenceHandle" />, or a handle where the <see cref="P:System.Reflection.Metadata.EntityHandle.IsNil" /> property is <see langword="true" />.</summary>
```

Comment:
this seems more appropriate for the value tag. Can we explain in plain English what this property is about?

Start with "Gets the..."

Reply:
```suggestion
        <summary>To be added.</summary>
```